torch/jit/__init__.py

```diff
@@ -354,22 +356,38 @@ def __getattr__(self, attr):
         return self.module._get_method(attr)
 
 
-def _script_graph(fn, _frames_up=0):
+def _compile_fn(fn, _frames_up=0):
+    mod = ScriptModule()
+    rcb = createResolutionCallback(_frames_up + 1)
+    ast = get_jit_ast(fn)
+    arg_types, ret_type = annotations.get_signature(fn, ast.num_params(), 0)
```

Comment:
This should parse into `FunctionSchema`, it is the reason we have that object. `get_signature` predated it, but now that FunctionSchema exists, we should use it. It will clean up a lot of the API calls that use it.